.github/workflows/merge-to-stage.js

```diff
@@ -218,7 +224,7 @@ const main = async (params) => {
     existingPRCount = body.match(/https:\/\/github\.com\/adobecom\/milo\/pull\/\d+/g)?.length || 0;
     console.log(`Number of PRs already in the batch: ${existingPRCount}`);
 
-    if (mergeLimitExceeded()) return console.log(`Maximum number of '${MAX_MERGES}' PRs already merged. Stopping execution`);
+    if (mergeLimitExceeded()) return console.log('Maximum number of PRs already merged. Stopping execution');
```

Comment:
As suggested, let's revert this change... Now that we add extra logic it's also even more important to know how many PRs we can even merge - hence that logging that is pretty useful